pipeline/tasks/gapic_tasks.py

```diff
@@ -134,7 +134,7 @@ def execute(self, language, api_name, final_repo_dir):
         # Such APIs don't need packman.
         if ((language == 'ruby' or language == 'nodejs') and
                 task_utils.is_output_gcloud(final_repo_dir)):
-            return
+            return final_repo_dir
```

Comment:
But this code can be simpler by having a single `return final_repo_dir` outside the conditional.